style.css

```diff
@@ -36,11 +36,70 @@ header {
   padding-left: 25px;
   padding-right: 25px;
   background-color: white;
+  align-items: center;
+}
+
+.mobile-nav {
+  position: absolute;
+  height: 100%;
+  min-width: 100%;
+  top: 0;
+  visibility: hidden;
+  background-color: #6070ff;
+  mix-blend-mode: multiply;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  z-index: 1;
+  margin: auto;
+  transition: all 400ms ease;
+  backdrop-filter: blur(4.5px);
```

Comment:
- [ ] **\[OPTIONAL\]** The mobile menu is working perfectly. However, consider changing the style to make the text stand out.

![image](https://user-images.githubusercontent.com/92215878/189975065-fc4d2f0e-dbd1-4d6e-83c7-7fcd45954bd3.png)